tfx/dsl/component/experimental/decorators_typeddict_test.py

```diff
@@ -140,8 +140,10 @@ def verify_beam_pipeline_arg(a: int) -> TypedDict('Output6', dict(b=float)):  #
 
 def verify_beam_pipeline_arg_non_none_default_value(
     a: int,
-    beam_pipeline: BeamComponentParameter[beam.Pipeline] = beam.Pipeline(),
+    beam_pipeline: BeamComponentParameter[beam.Pipeline] = 0,
```

Comment:
ditto

Reply:
Error:
`
FAILED tfx/dsl/component/experimental/decorators_typeddict_test.py::ComponentDecoratorTest::testBeamPipelineDefaultIsNotNoneFails - TypeError: Argument a should be a Channel of type <class 'tfx.types.standard_artifacts.Integer'> (got 1)
`

Reply:
same here

Reply:
Updated